<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="voltagePlot" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>921</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="output_enable">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Enable</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>701</width>
       <height>137</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="Offset_column">
        <item>
         <widget class="QLineEdit" name="Offset">
          <property name="text">
           <string>Offset</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="offset_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Frequency_column">
        <item>
         <widget class="QLineEdit" name="Frequency">
          <property name="text">
           <string>Frequency</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="frequency_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Amplitude_column">
        <item>
         <widget class="QLineEdit" name="Amplitude">
          <property name="text">
           <string>Amplitude</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="amplitude_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Mode_select">
        <item>
         <widget class="QRadioButton" name="sine_wave_button">
          <property name="text">
           <string>sine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="square_wave_button">
          <property name="text">
           <string>square</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="triangle_wave_button">
          <property name="text">
           <string>triangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dc_output_button">
          <property name="text">
           <string>dc output</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="Mode">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="output_voltage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_voltage_text">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Voltage</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
